docs/getting-started/linux-manual.md

```diff
@@ -160,11 +163,44 @@ $ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_REALM=us0 \
 
 > Use of a SemVer tag over `latest` is highly recommended.
 
-In the case of Docker, a volume mount may be required to load custom
-configuration as shown above.
+In the case of Docker, a volume mount may be required to load the custom
+configuration file as shown above.
 
 If the custom configuration includes a `memory_limiter` processor then the
 `ballast_size_mib` parameter should be the same as the
 `SPLUNK_BALLAST_SIZE_MIB` environment variable. See
 [gateway_config.yaml](../../cmd/otelcol/config/collector/gateway_config.yaml)
 as an example.
+
+In order to use `SPLUNK_CONFIG_YAML`, the configuration YAML needs to be 
```

Comment:
It could be helpful to show something like `cat agent_config.yaml | awk 1 ORS='\\n'` for how they could grab this manually.